docs/swagger.yaml

```diff
@@ -19,24 +19,28 @@ paths:
             type: string
             default: "{{vipps_system_name}}"
             example: Acme Commerce
+            maxLength: 30
         - name: Vipps-System-Version
           in: header
           schema:
             type: string
             default: "{{vipps_system_version}}"
             example: "3.1.2"
+            maxLength: 30
         - name: Vipps-System-Plugin-Name
           in: header
           schema:
             type: string
             default: "{{vipps_system_plugin_name}}"
             example: acme-webshop
+            maxLength: 30
```

Comment:
The thought is; This way, people will get notification that it's too long, and we can maybe tighten up from 35 to 30 chars backend ?

Reply:
Yeah, basically pretend the max is 30